start/system/pom.xml

```diff
@@ -35,12 +35,12 @@
         </dependency>
         <dependency>
             <groupId>io.openliberty.features</groupId>
-            <artifactId>mpRestClient-1.1</artifactId>
```

Comment:
Same comment but for the `start/` directory.

Reply:
The test may require it

Reply:
addressed by #67